apps/api/src/deployment/routes/deployments/deployments.router.ts

```diff
@@ -0,0 +1,71 @@
+import { createRoute } from "@hono/zod-openapi";
+import { container } from "tsyringe";
+import { z } from "zod";
+
+import { OpenApiHonoHandler } from "@src/core/services/open-api-hono-handler/open-api-hono-handler";
+import { DeploymentController } from "@src/deployment/controllers/deployment/deployment.controller";
+import { CreateDeploymentRequestSchema, CreateDeploymentResponseSchema, GetDeploymentParamsSchema, GetDeploymentResponseSchema } from "@src/deployment/http-schemas/deployment.schema";
+
+const getRoute = createRoute({
+  method: "get",
+  path: "/v1/deployments/{dseq}",
```

Comment:
`dseq` is not a unique identifier for a deployment. `userId` + `dseq` or `address/owner` + `dseq` is. In our case user seems to be more appropriate. So the interface should prob be something like:

```suggestion
  path: "/v1/deployments/${userId}/{dseq}",
```

It's not perfect so it's totally open to discussion, however user should be taken into account